samples/contrib/aws-samples/debugger_component_demo/debugger-component-demo.py

```diff
@@ -73,78 +72,30 @@
     'num_round': '50'
 }
 
-channelObjList = []
-
-channelObj = {
-    'ChannelName': '',
-    'DataSource': {
-        'S3DataSource': {
-            'S3Uri': '',
-            'S3DataType': 'S3Prefix',
-            'S3DataDistributionType': 'FullyReplicated'
-        }
-    },
-    'ContentType': 'text/csv',
-    'CompressionType': 'None',
-    'RecordWrapperType': 'None',
-    'InputMode': 'File'
-}
-
-channelObj['ChannelName'] = 'train'
-channelObj['DataSource']['S3DataSource']['S3Uri'] = 's3://kubeflow-pipeline-data/mnist_kmeans_example/input/valid_data.csv'
-channelObjList.append(copy.deepcopy(channelObj))
-
 
 @dsl.pipeline(
-    name='xgboost-mnist-debugger',
+    name='XGBoost Training Pipeline with bad hyperparameters',
     description='SageMaker training job test with debugger'
 )
-def training(
+def training(role_arn="", bucket_name=""):
```

Comment:
nit: Fill in a bad default `bucket_name` (`my-bucket` is our go-to) so that it will be a valid training job, but fail with validating. This makes it easier to debug.